blueledger-webapp/services/expenses/expense-mapper.ts

```
@@ -0,0 +1,45 @@
+
+  static toApiRequest(data: ExpenseFormData): ExpenseApiRequest {
+    return {
+      ...data,
+      date: data.date.toISOString(),
+    };
+  }
+
+  static toTypeList(apiResponses: ExpenseApiResponse[]): ExpenseType[] {
+    return apiResponses.map(this.toType);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)